app-text/robodoc/Manifest

```diff
@@ -1 +1,3 @@
 DIST robodoc-4.99.36.tar.gz 280680 BLAKE2B 94662733945f6bbed5a5b6207423c175b1ce0c700f3488cc55bbd79d864c0ee10d49ac27ab8869c09a06708d31e6d1f9e7600bb0fdd7e0477559e19b21ec0a30 SHA512 6643cb1419c58195f26899e629a4a7470014b425e05b7615e523f972b67eb0e39421c373b8c80b53a4dd0d121d61481a7a0e66f7fca7fe82422d56b8ff048b9b
+DIST robodoc-4.99.41.tar.gz 299656 BLAKE2B 88eeeb52d519b999bc85c2a2c696fea81de88c5ef848188854c24dbcb45fd2a6ca650a0ed2e2927f5631b2c82f2c797b09e97879675bbcaefc6c43097211b0fa SHA512 08d94bc679dc186ae30acf14db215fb8b9e572b0a54e71f2f0c3aa6b9c585674ab90cc610f676de17533ae7dad75599cc74e761fe1f3d24d378d8e74bce40833
```

Comment:
Is there a reason you add two versions? Is someone going to install 4.99.41, if 4.99.43 will be available too? Also 4.99.36 isn't stabilized so consider removing it after bumping to latest version, unless there is a good reason to keep it.

Reply:
I've kept only the latest stable release as of April 2015 (version 4.99.43).
There was no specific "robodoc" reason for the multiple versions, but rather practice for me since this was amongst my first PRs.